lib/mpl_toolkits/axisartist/axislines.py

```diff
@@ -184,12 +178,13 @@ def get_line(self, axes):
 
 class FixedAxisArtistHelperRectilinear(_FixedAxisArtistHelperBase):
 
+    @_api.delete_parameter("3.9", "nth_coord")
     def __init__(self, axes, loc, nth_coord=None):
         """
         nth_coord = along which coordinate value varies
         in 2D, nth_coord = 0 ->  x axis, nth_coord = 1 -> y axis
         """
-        super().__init__(loc, nth_coord)
+        super().__init__(loc)
```

Comment:
Doesn’t this make the parameter unused, which means we silently accept anything?

Reply:
But passing *any* value for nth_coord will now raise a DeprecationWarning anyways.  (If you really want, I could duplicate the check here that nth_coord, if passed, is consistent with loc, but I think that's overkill.  In any case I cannot forward it to the super class as that would also raise a DeprecationWarning.)

Reply:
Fair enough.